ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/router_bgp.py

```diff
@@ -127,6 +127,9 @@ def router_bgp(self) -> dict | None:
                     "bfd": get(link, "bfd"),
                 }
 
+                if self.shared_utils.shutdown_bgp_towards_undeployed_peers is True and link["peer_is_deployed"] is False:
```

Comment:
You should also shutdown any overlay peerings to this device.

Reply:
Added